docs/contributing.rst

```diff
@@ -182,145 +226,115 @@ Now that you have a general sense of how the repo is organized (and how you will
   :align: center
   :alt: decorative line
 
-1. First, you will want to **render the handbook locally** so that you can get a sense of the impact of the changes you make on the website.
+1. First, make sure you have **activated your conda environment** and are working in the correct directory: 
 
 .. code-block:: bash
 
     # change directory to be in the outermost folder of the repo
     cd ~/Desktop/princeton_handbook/handbook
+
     # activate the conda environment
     conda activate princeton_handbook
-    # render the handbook locally
-    make build
-
-* The output on your terminal will look something like the screenshot below, with the location of the html pages living in the location circled in blue. 
-
-.. image:: images/make_build_output.png
-  :width: 400px
-  :align: center
-  :alt: output you get after 'make build'
-  :class: with-border
-
-* Open up Finder and double click on :blue:`index.html`, which will be in the directory :blue:`handbook/docs/_build/html` as discussed in the previous section. This will open a local version of the handbook in your internet browser.
 
 .. image:: images/decorative_line4.png
   :width: 700px
   :height: 8px
   :align: center
   :alt: decorative line
 
-2. You only have to do this ONCE: **Add the official handbook repository as the upstream remote**.
+2. **Pull any new commits in the official handbook repo** to keep your master branch up to date:
 
 .. code-block:: bash
 
-    # add official repo as the upstream remote
-    git remote add upstream https://github.com/brainhack-princeton/handbook
-
-* :red:`Why are we doing this?` We want your local directory to point to two different remotes (or places on Github where code is stored). The first one should be the **origin**, which should point to your forked repo on Github. The second one should be the **upstream** branch, which should point to the official handbook repo on Github. Having this organization will allow you to pull changes from the upstream branch so that you can make edits on the most up-to-date version of the repo, while allowing you the flexibility to do work on your repo. To check that all of this holds true, run the following commands on your terminal:
+    # fetches and merges the remote changes to your local files
+    git pull --ff-only
 
-.. code-block:: bash
-    
-    # view the remotes you currently have
-    git remote
-    # view the url of the git repo the origin remote points to
-    git remote get-url origin
-    # view the url of the git repo the upstream remote points to
-    git remote get-url upstream
+    # push the commits to your forked repo
+    git push origin master
 
-.. image:: images/looking_at_remotes.png
-  :width: 700px
-  :align: center
-  :alt: looking at remotes
+* You have to do ``git push origin master`` in order to push any commits that you pulled from the official repo (**upstream**) to your forked repo (**origin**). 
```

Comment:
I'm not sure this strictly necessary in all cases; might want to soften the wording a bit, more like "If your forked repo (on GitHub) gets behind the original repo, do X"

Reply:
I'll add a note about this...